### PR TITLE
Android: update to zxing 3.3.0

### PR DESCRIPTION
**Pull request by m1ga**
* Updated ZXing
* Moved all the log.i to log.d
* frameHeight/frameWidth as an option to set the capture region

### CODE REVIEW

**Comment by hansemannn** [comment]
Moin Michael! Do you plan to finish this PR soon? :-)

**Comment by m1ga** [comment]
I'll have some spare time this weekend so I can start fixing the last bugs! Front camera is currently missing and need to test/optimize some frameRect parts.

**Comment by m1ga** [comment]
* Front-Camera support added (will optimize it a bit more)
* moved all the log.i to log.d 
* if no frameHeight/frameWidth is set it will automatically add a proper rectangle again

*Todo*
* I want to optimize the camera change a bit
* remove some unused parts (e.g. History-Manager?)
* some more testing

File: android/src/ti/barcode/FrontCamera.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+			Log.w("BarcodeModule", "No cameras!");
+			return -1;
+		}
+
+		while (index < numCameras) {
+			Camera.CameraInfo cameraInfo = new Camera.CameraInfo();
+			Camera.getCameraInfo(index, cameraInfo);
+			CameraFacing reportedFacing = CameraFacing.values()[cameraInfo.facing];
+			if (reportedFacing == CameraFacing.FRONT) {
+				break;
```
Comment (hansemannn) [inline android/src/ti/barcode/FrontCamera.java]: Why not just return here? and return -1 in all other cases?

Comment (m1ga) [inline android/src/ti/barcode/FrontCamera.java]: You are right, updated :-)

**Comment by m1ga** [comment]
Feel free to test it now.

For future updates it would be good to use a third-party library (e.g. https://github.com/journeyapps/zxing-android-embedded) that handles the zxing parts and you can add it as an activity or integrate it as a view (to add overlays, use it in non-fullscreen mode,..).

**Comment by hansemannn** [comment]
LGMT, thank you!

**Pull request merged**